spec/models/group_assignment_spec.rb

```diff
@@ -44,6 +44,22 @@
     end
   end
 
+  describe 'exclusion of reserved words from title' do
+    let(:organization) { create(:organization) }
+
+    it 'ensures RecordInvalid error when GroupAssignment title is \'new\'' do
+      expect do
+        create(:group_assignment, organization: organization, title: 'new')
+      end.to raise_error(ActiveRecord::RecordInvalid, 'Validation failed: Your assignment title is reserved')
+    end
+
+    it 'ensures RecordInvalid error when GroupAssignment title is \'update\'' do
+      expect do
+        create(:group_assignment, organization: organization, title: 'update')
+      end.to raise_error(ActiveRecord::RecordInvalid, 'Validation failed: Your assignment title is reserved')
+    end
+  end
```

Comment:
I think this can be made into a single test.

```ruby
describe 'exclusion of reserved words from exclusion from title' do
  let(:organization) { create(:organization) }
  it 'will not save if using an action_method as the title' do
    GroupAssignmentsController.action_methods.to_a.each do |action_method|
      expect do
        create(:group_assignment, organization: organization, title: action_method)
      end.to raise_error(ActiveRecord::RecordInvalid, 'Validation failed: Your assignment title is reserved')
    end  
  end
end
```